examples/authProvider/googleLogin/src/interfaces/google.d.ts

```diff
@@ -3,15 +3,15 @@ interface IdConfiguration {
     auto_select?: boolean;
     callback: (handleCredentialResponse: CredentialResponse) => void;
     login_uri?: string;
-    native_callback?: (...args: any[]) => void;
+    native_callback?: (...args: any[]) => void; // eslint-disable-line
```

Comment:
```suggestion
    native_callback?: (...args: unknown[]) => void;
```

`unknown` is a better fix imo

Reply:
done